modules/javafx.graphics/src/main/java/javafx/scene/text/Text.java

```diff
@@ -1088,14 +1088,32 @@ public final PathElement[] caretShape(int charIndex, boolean caretBias) {
 
     /**
      * Returns the shape for the range of the text in local coordinates.
+     * <p>
+     * NOTE: this shapes returned do not include line spacing.
```

Comment:
I would remove the "NOTE:", making it a simple statement (and probably no need for the `<p`> tag), since this is just what the method does.

Question: should this method be deprecated to be consistent with TextFlow?